app/client/src/components/DropZone.vue

```diff
@@ -8,7 +8,7 @@
     <p> Uploaded files:</p>
     <!--only adding this temporarily to have something testable for e2e tests-->
     <div class="uploaded-info"><p v-for="file in results.perIsolate" :key="file.hash">
-      {{file.filename}} {{file.hash}} {{file.amr}} {{file.sketch}}
+      {{file.filename}} {{file.hash}} {{file.amr.slice(2, 10)}} {{file.sketch.slice(2, 4)}}
```

Comment:
These numbers are a bit magic too - rather than doing the slicing here, could you make methods to do that, with some explanatory comments

Reply:
This is actually just meant for testing. When I have the final table representation of the results I won't need this anymore, just have it for now to have something testeable on the page that represents what's stored in state. I'm using the .slice() since the actual sketch and amr results are too verbose to print completely